src/app/[lang]/(main)/spending/national-defence/page.tsx

```diff
@@ -39,7 +39,6 @@ export default async function Department(props: PageLangParam) {
 	const lang = (await props.params).lang;
 	initLingui(lang);
 
-	const { t } = useLingui();
```

Comment:
`42:10  Error: 't' is assigned a value but never used.  @typescript-eslint/no-unused-vars`